components/brave_news/common/brave_news.mojom

```diff
@@ -175,10 +175,14 @@ struct Cluster {
   array<ArticleElements> articles;
 };
 
+// Placeholder FeedV2 Ad - the actual ad is fetched dynamically when the item
+// comes into the viewport.
+struct FeedV2Ad {};
+
 union FeedItemV2 {
   Article article;
   HeroArticle hero;
-  PromotedArticle advert;
+  FeedV2Ad advert;
```

Comment:
This should only be a placeholder, as we want to dynamically load the ads as you scroll through the feed